docs/semgrep-ci/configuration-reference.md

```diff
@@ -69,17 +69,25 @@ To find more details about some of these configuration options, see the followin
 
 ## Diff-aware scanning (`SEMGREP_BASELINE_REF`)
 
-For [diff-aware scans](overview.md#features), this option filters scan results to those introduced after the git commit, in a branch, or tag. For example, you have a repository with 10 commits. You set the commit number 8 as the baseline. Consequently, Semgrep only returns scan results introduced by changes in commits 9 and 10.
+Semgrep scans can be classified by scope. The scope of a scan refers to what lines of code are scanned in a codebase. When classifying scans by scope, there are two types of scans:
+
+<dl>
+    <dt>Full scan</dt>
+    <dd>A full scan will scan your entire codebase and report every finding in the codebase. It is recommended to perform a full scan of your <code>main</code> branch at a regular cadence, such as every night or every week, depending on your own needs. This ensures that Semgrep App has a full list of all findings in your code base, regardless of when they were introduced. To run a full scan, simply run <code>semgrep ci</code> without setting <code>SEMGREP_BASELINE_REF</code> environment variable.</dd>
```

Comment:
- Do not use will in technical writing. Only use will when you are referring to the time that needs to elapse between action and goal. See https://developers.google.com/style/tense?hl=en.
- Since I removed `will`, I am trying to avoid `scan scans` in the first sentence.

```suggestion
    <dd><p>A full scan is scanning your entire codebase and reports every finding in the codebase. It is recommended to perform a full scan of your <code>main</code> branch at a regular cadence, such as every night or every week. This ensures that Semgrep App has a full list of all findings in your code base, regardless of when they were introduced. To run a full scan, run <code>semgrep ci</code> without setting <code>SEMGREP_BASELINE_REF</code> environment variable.</p></dd>
```

Reply:
Honestly, "is scanning" is not better than "will scan".

I think will is fine because this is not a procedure. If the user chooses to use a full scan, then they will scan their codebase. Does that make sense?

Reply:
One of the reasons why we do this is that some engineers are using automatic translations of pages and some languages do not have future tense (so will cause a bit of havoc). @s-santillan See Google style guide suggestion "However, the fact that the reader will be writing and running code in the future isn't a good reason to use future tense. Stick with present tense where possible, as shown in the previous Recommended example." From my experience, TWs are only allowed to use "will" really when you are talking about things such as "The hedgehog will hibernate starting in October." (actual specific future). It might be ok in this case, but I would avoid it here also. I will leave this decision up to you. (Please stick with the division to paragraphs I suggest.)

Reply:
I think I'll go with "A full scan runs on your entire codebase..."